ERCS/erc-7887.md

```diff
@@ -0,0 +1,378 @@
+---
+eip: 7887
+title: Cancelation for ERC-7540 Tokenized Vaults
+description: Extension of ERC-7540 with cancelation support
+author: Jeroen Offerijns (@hieronx), Vikram Arun (@vikramarun)
+discussions-to: https://ethereum-magicians.org/t/erc-7887-cancelation-for-erc-7540-tokenized-vaults/22906
+status: Draft
+type: Standards Track
+category: ERC
+created: 2025-02-18
+requires: 165, 7540
+---
+
+## Abstract
+
+The following standard extends [ERC-7540](./eip-7540.md) by adding support for asynchronous cancelation flows.
```

Comment:
Your abstract should provide enough information to the reader to give them a high level (but still technical) of _how_ your proposal accomplishes its goals. You could describe in a bit more detail what an "asynchronous cancelation flow" is, and maybe what functions enable it?